service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/PartyAddedToCaseContentProvider.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences;
+import uk.gov.hmcts.reform.fpl.service.DateFormatterService;
+import uk.gov.hmcts.reform.fpl.service.HearingBookingService;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.PARTY_ADDED_TO_CASE_THROUGH_DIGITAL_SERVICE_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.EMAIL;
+
+@Service
+public class PartyAddedToCaseContentProvider extends AbstractEmailContentProvider {
+
+    private final PartyAddedToCaseByEmailContentProvider partyAddedToCaseEmailContentProvider;
+    private final PartyAddedToCaseThroughDigitalServiceContentProvider
+        partyAddedToCaseThroughDigitalServicelContentProvider;
+
+    @Autowired
+    public PartyAddedToCaseContentProvider(@Value("${ccd.ui.base.url}") String uiBaseUrl,
+                                           DateFormatterService dateFormatterService,
+                                           HearingBookingService hearingBookingService,
+                                           PartyAddedToCaseByEmailContentProvider partyAddedToCaseEmailContentProvider,
+                                           PartyAddedToCaseThroughDigitalServiceContentProvider
+                                                   partyAddedToCaseThroughDigitalServicelContentProvider) {
+        super(uiBaseUrl, dateFormatterService, hearingBookingService);
+        this.partyAddedToCaseEmailContentProvider = partyAddedToCaseEmailContentProvider;
+        this.partyAddedToCaseThroughDigitalServicelContentProvider
+            = partyAddedToCaseThroughDigitalServicelContentProvider;
+    }
+
+    public Map<String, Object> getPartyAddedToCaseNotificationParameters(CaseDetails caseDetails,
+                                        RepresentativeServingPreferences servingPreferences) {
+        if (servingPreferences == EMAIL) {
+            return partyAddedToCaseEmailContentProvider
+                .buildPartyAddedToCaseNotification(caseDetails);
+        } else {
+            return partyAddedToCaseThroughDigitalServicelContentProvider
+                .buildPartyAddedToCaseNotification(caseDetails);
```

Comment:
Could `partyAddedToCaseEmailContentProvider` and `partyAddedToCaseThroughDigitalServicelContentProvider` not be the same service. Just exposing two different methods?

It means we could further extract 

```
return Map.of(
  "firstRespondentLastName", getFirstRespondentLastName(caseData.getRespondents1()),
  "familyManCaseNumber", defaultIfNull(caseData.getFamilyManCaseNumber(), "")
);
```

Into a more reusable private method.